src/rgw/rgw_process.cc

```diff
@@ -225,6 +225,8 @@ int process_request(rgw::sal::RGWRadosStore* const store,
                                                frontend_prefix,
                                                client_io, &mgr, &init_error);
   rgw::dmclock::SchedulerCompleter c;
+  const auto& lua_package_path = g_conf().get_val<std::string>("rgw_luarocks_location"); 
```

Comment:
it would be nice if we could cache this somehow...either for the lifetime of radosgw, or the cached config option mechanism that has been around for a while

Reply:
@mattbenjamin caching as a data member to `RGWRadosStore`.  here: https://github.com/ceph/ceph/pull/37924/commits/7ff10c504a7e015967873aa9dddbda84f03fe786#diff-9dec70d3aeed008aebebd862985b5d53466e1b55bdd6220b984ab099a9a29354R414
does it make sense?